frontend/integration-tests/tests/kubevirt/utils/utils.ts

```diff
@@ -47,16 +40,37 @@ export function removeLeakedResources(leakedResources: Set<string>) {
   leakedResources.clear();
 }
 
+export function addLeakableResource(leakedResources: Set<string>, resource) {
+  leakedResources.add(
+    JSON.stringify({name: resource.metadata.name, namespace: resource.metadata.namespace, kind: resource.kind})
+  );
+}
+
+export function removeLeakableResource(leakedResources: Set<string>, resource) {
+  leakedResources.delete(
+    JSON.stringify({name: resource.metadata.name, namespace: resource.metadata.namespace, kind: resource.kind})
+  );
+}
+
+export function createResource(resource) {
+  execSync(`echo '${JSON.stringify(resource)}' | kubectl create -f -`);
+}
+
 export function createResources(resources) {
   resources.forEach(resource => {
-    execSync(`echo '${JSON.stringify(resource)}' | kubectl create -f -`);
+    createResource(resource);
   });
 }
 
+export function deleteResource(resource) {
+  const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
+  execSync(`kubectl delete -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`);
+}
+
 export function deleteResources(resources) {
   resources.forEach(resource => {
-    const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
-    execSync(`kubectl delete -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`);
+    resource.kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
```

Comment:
why is this line needed here? Isn't this also executed in `deleteResource` ?

Reply:
Is this what you mean in the previous comment? `resources.forEach(resource => createResource(resource));`

And yes here it is not necessary.

Reply:
you mean the lamdas? 
```js
resources.forEach(createResource)
```

Reply:
Ah, cool, didn't realize it works like that. Done.